aptos-move/aptos-vm/src/block_executor/vm_wrapper.rs

```diff
@@ -53,29 +53,77 @@ impl<'a, S: 'a + StateView + Sync> ExecutorTask for AptosExecutorTask<'a, S> {
     }
 
     // This function is called by the BlockExecutor for each transaction is intends
-    // to execute (via the ExecutorTask trait). It can be as a part of sequential
-    // execution, or speculatively as a part of a parallel execution.
-    fn execute_transaction(
+    // to execute (via the ExecutorTask trait). This function is run speculatively
+    // as a part of a parallel execution.
+    fn execute_transaction_parallel(
+        &self,
+        view: &impl StateView,
+        txn: &PreprocessedTransaction,
+        txn_idx: TxnIndex,
+        aggregator_enabled: bool,
+    ) -> ExecutionStatus<AptosTransactionOutput, VMStatus> {
+        let log_context = AdapterLogSchema::new(self.base_view.id(), txn_idx as usize);
+        match self.vm.execute_single_transaction(
+            txn,
+            &view.as_move_resolver(),
+            &log_context,
+            aggregator_enabled,
+        ) {
+            Ok((vm_status, vm_output, sender)) => {
+                // If aggregators are disabled for user transactions, then delta_change_set should be empty as there should not be any delta outputs.
```

Comment:
nit: split over multiple lines? I am not sure how many we use for our code (it seems quite random) but I usually use 80

Reply:
Done.